test-suite/js/src/test/scala/org/scalajs/testsuite/library/ObjectTest.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.Platform
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.{ JSBracketAccess, JSName }
+
+class ObjectTest {
+  import ObjectTest._
+
+  @Test def getOwnPropertyDescriptors(): Unit = {
+    assume(Platform.executingInNodeJS)
```

Comment:
Please `assumeTrue` declared in `org.junit.Assume._` for unit test assumptions. Scala's `Predef.assume` is not guaranteed to be interpreted as a JUnit assumption.

Reply:
Fixed in https://github.com/scala-js/scala-js/pull/3869/commits/6e0316580f91ff1133c5ab102ecde13180778571